lnwallet/channel.go

```diff
@@ -565,40 +565,189 @@ type htlcView struct {
 	theirUpdates []*PaymentDescriptor
 }
 
-// fetchHTLCView returns all the candidate HTLC updates which should be
-// considered for inclusion within a commitment based on the passed HTLC log
-// indexes.
-func (lc *LightningChannel) fetchHTLCView(theirLogIndex, ourLogIndex uint32) *htlcView {
+// getHTLCView convert HTLC logs into htlcView representation.
+func (lc *LightningChannel) getHTLCView() *htlcView {
```

Comment:
With this change, this function no longer really returns an HTLC view. Instead, it just returns both lists of active log entries without doing any sort of filtration. Therefore, this function can be altered to simply return to slices of the local and remote updates.